apidoc/Titanium/UI/WebViewConfiguration.yml

```diff
@@ -0,0 +1,107 @@
+    user can select, and many other options. This property can only be set when creating the webview and will be ignored when set afterwards.
+
+    E.g.
+
+    var config = Ti.UI.createWebViewConfiguration({
+        allowsPictureInPictureMediaPlaback: true
+    });
+
+    var webView = WK.createWebView({
+        configuration: config
+    });
+
+extends: Titanium.Proxy
+since: "7.4.0"
+platforms: [iphone, ipad]
+
+properties:
+
+  - name: preferences
+    summary: The preference settings to be used by the web view.
+    type: PreferencesObject
+    platforms: [iphone, ipad]
+
+  - name: selectionGranularity
+    summary: |
+        The level of granularity with which the user can interactively select content in the web view.
+    type: Number
+    platforms: [iphone, ipad]
+    constants: Titanium.UI.SELECTION_GRANULARITY_*
+    default: Titanium.UI.SELECTION_GRANULARITY_DYNAMIC
+
+  - name: mediaTypesRequiringUserActionForPlayback
+    summary: Determines which media types require a user gesture to begin playing.
+    type: Number
+    constants: Titanium.UI.AUDIOVISUAL_MEDIA_TYPE_*
+    osver: {ios: {min: "10.0"}}
+    platforms: [iphone, ipad]
+    default: Titanium.UI.AUDIOVISUAL_MEDIA_TYPE_NONE
+
+  - name: suppressesIncrementalRendering
+    summary: |
+        A Boolean value indicating whether the web view suppresses content rendering until it is fully 
+        loaded into memory.
+    type: Boolean
+    platforms: [iphone, ipad]
+    default: false
+
+  - name: allowsInlineMediaPlayback
+    summary: |
+        A Boolean value indicating whether HTML5 videos play inline or use the native full-screen controller.
+    description: |
+        You must set this property to play inline video. Set this property to true to play videos inline. 
+        Set this property to false to use the native full-screen controller. When adding a video element 
+        to a HTML document on the iPhone, you must also include the playsinline attribute.
+        The default value for iPhone is false and the default value for iPad is true.
+    type: Boolean
+    platforms: [iphone, ipad]
+    default: For iPhone false and for iPad true.
+
+  - name: allowsAirPlayMediaPlayback
+    summary: A Boolean value indicating whether AirPlay is allowed.
+    type: Boolean
+    platforms: [iphone, ipad]
+    default: true
+
+  - name: allowsPictureInPictureMediaPlayback
+    summary: A Boolean value indicating whether HTML5 videos can play picture-in-picture.
+    type: Boolean
+    platforms: [iphone, ipad]
+    default: true
+
+  - name: processPool
+    summary: The process pool from which to obtain the  Web Content process of view.
+    description: |
+        When a web view is initialized, either a new web content process is created for it from the 
+        specified pool or an existing process in that pool is used.
+    type: Titanium.UI.WebViewProcessPool
+    platforms: [iphone, ipad]
```

Comment:
All of these are missing the minimum iOS version.

Reply:
@hansemannn Should we really mention iOS version, which is iOS 8.0? As we are going to support these with SDK 8.0.0+ and min iOS version for that will be 9.0.